lib/topological_inventory/orchestrator/worker.rb

```diff
@@ -162,10 +162,13 @@ def authentication_with_password(id, tenant_account)
       def collector_definitions
         @collector_definitions ||= begin
           {
-            "amazon"    => {
+            "amazon"        => {
               "image" => "topological-inventory-amazon:#{collector_image_tag}"
             },
-            "openshift" => {
+            "ansible_tower" => {
```

Comment:
This needs to match https://github.com/ManageIQ/sources-api/blob/master/db/seeds.rb#L46 I think